pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -158,8 +161,39 @@ var (
 			secret.Name = strings.ReplaceAll(secret.Name, hash, "<hash>")
 		},
 	}
+	RewriteRefreshPeriod = &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			secret.Annotations = humanizeRefreshPeriodFromMetadata(secret.Annotations)
+		},
+		rewriteConfigMapFn: func(configMap *corev1.ConfigMap) {
+			configMap.Annotations = humanizeRefreshPeriodFromMetadata(configMap.Annotations)
+		},
+	}
 )
 
+func humanizeRefreshPeriodFromMetadata(annotations map[string]string) map[string]string {
+	result := map[string]string{}
+	if annotations == nil {
+		return result
+	}
+	for k, v := range annotations {
+		result[k] = v
+	}
+	period, ok := annotations[certrotation.CertificateRefreshPeriodAnnotation]
+	if !ok {
+		return result
+	}
+	d, err := time.ParseDuration(period)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to parse rewrite period %s: %v\n", period, err)
+		return result
+	}
+	humanReadableDate := durationToHumanReadableString(d)
+	result[certrotation.CertificateRefreshPeriodAnnotation] = humanReadableDate
+	result[rewritePrefix+"HumanizeRewritePeriod"] = period
```

Comment:
I think that this should be `RewriteRefreshPeriod` now.

Reply:
Fixed